### PR TITLE
Updated data.sql, queries.sql, schema.sql and Screenshots folder.

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the following was done: 

- Species_id in animals table was updated
- Foreign Keys were inserted
- Various data queries were made. 

Screenshots
Screenshot 1
Screenshot 2 1
Screenshot 2 2
Screenshot 2 3
Screenshot 3 1
Screenshot 3 2
Screenshot 3 3
Screenshot 3 4
Screenshot 4 1
Screenshot 4 2
Screenshot 5 1
Screenshot 4 3

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -77,6 +77,51 @@ FROM animals
...
+SELECT animals.name AS digimon_name
+FROM animals
+JOIN species ON animals.species_id = species.id
+JOIN owners ON animals.owner_id = owners.id
+WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
+
+SELECT a.name AS animal_name
+FROM animals AS a
+JOIN owners AS o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester';
```
Comment (gemmen29) [inline queries.sql]: - [ ] Kindly fix this query to match this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=List%20all%20animals%20owned%20by%20Dean%20Winchester%20that%20haven%27t%20tried%20to%20escape.). You listed all animals owned by Dean Winchester but you didn't implement `that haven't tried to escape` in the requirement.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @dvcorreia1984, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (dvcorreia1984) [reply queries.sql]: Ok, thanks for this!

**Review by Strangeal — APPROVED** [review]
Hi @dvcorreia1984, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Added review changes ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**